docs/index.adoc

```diff
@@ -92,36 +92,56 @@ Choose from your development language / framework and follow the links for the i
 |===================================
 ||Java|JavaScript|C#|Dart|Go
 |Documentation| link:https://github.com/featurehub-io/featurehub/tree/master/sdks/java/client-java-jersey[Java-Jersey]|link:https://github.com/featurehub-io/featurehub/tree/master/sdks/typescript/client-typescript-core[Javascript, Typescript, Node, React, Angular]|https://github.com/featurehub-io/featurehub/tree/master/sdks/client-csharp/FeatureHubSDK[C#]|link:https://github.com/featurehub-io/featurehub/tree/master/sdks/sdks/dart/client-dart-sdk[Dart]|https://github.com/featurehub-io/featurehub/tree/master/sdks/client-go[Go]
-|Examples|link:https://github.com/featurehub-io/featurehub/tree/master/examples/todo-backend-java[Java-Jersey example] | link:https://github.com/featurehub-io/featurehub/tree/master/examples/todo-backend-typescript[Node server example] , link:../examples/todo-frontend-react-typescript/[React app example] |Coming soon!|Coming soon!|Coming soon!
+|Examples|link:https://github.com/featurehub-io/featurehub/tree/master/examples/todo-backend-java[Java-Jersey example] | link:https://github.com/featurehub-io/featurehub/tree/master/examples/todo-backend-typescript[Node server example] , link:../examples/todo-frontend-react-typescript/[React app example] |https://github.com/featurehub-io/featurehub/tree/master/sdks/client-csharp/ConsoleApp1[C# Console]|https://github.com/featurehub-io/featurehub/tree/master/sdks/dart/client-dart-sdk/example/dart_cli[Dart Server], https://github.com/featurehub-io/featurehub/tree/master/sdks/dart/client-dart-sdk/example/web_example[Flutter example]|Coming soon!
 |===================================
 
 === Capabilities overview
 
 This overview seeks to indicate the capabilities of the SDKs and explain what they are and what the do.
 If you are considering helping us by writing a new SDK for your favourite language, or expand on an existing library, this table of capability indicates what each different language can support and where extra work is helpful.
 
+=== General Run Capabilities for SDKs
 [options="header"]
 |===================================
-|Capability|Java|Javascript^1^|Go|Dart^2^|C#
+|Runtime Capability|Java|Javascript^1^|Go|Dart^2^|C#
 |Event Streaming|Y|Y|Y|Y|Y
 |Background Start|Y|Y|Y|Y|Y
 |Block until Start|N|N|Y|N|N
 |Readyness Listeners|Y|Y|Y|Y|Y
 |Feature Listeners|Y|Y|Y|Y|Y
 |Feature Listener Removal|N|N|Y|Y|Y
-|Catch & Release|N|Y|N|Y|N
 |Analytics Support|Y|Y|N|Y|N
 |Google Analytics|Y|Y|N|N|N
+|===================================
+
+==== UI focused capabilities for SDKs
```

Comment:
maybe better to say Web focused?

Reply:
it is used in Dart as well, which reminds me!